modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -90,12 +87,9 @@ object types {
   object RewardsConfig {
 
     val default = RewardsConfig(
-      epochs = 4,
-      epochDurationInYears = 2.5,
-      baseEpochReward = Amount(85333333320000000L),
       softStaking = SoftStakingAndTestnetConfig(
         softStakeAddress = Address("DAG77VVVRvdZiYxZ2hCtkHz68h85ApT5b2xzdTkn"),
-        softStakeCount = 0L,
+        softStakeCount = 20L,
```

Comment:
I set it to some value to be sure that it returns `> 0 ` for soft staking numerator. Maybe we should test against arbitrary values of different node counts instead of hardcoding it.

Reply:
Actually, it's not a bad idea. If you decide to do it, please check for the `@derive(arbitrary)` to derive an Arbitrary instance for config generator.